.cargo/config

```diff
@@ -1,4 +1,8 @@
-[build]
 # Informs the linker that some of the symbols for Postgres won't be available
-# until runtime on the dynamic library load.
+# until runtime on the dynamic library load. Flags differ based on OS family
```

Comment:
To nitpick, flags differ based on the linker used; on Windows, the MSVC `link.exe` is used.

Reply:
Thanks for the clarification (I'm not familiar with C compialtion). The reason I put "based on OS family" is because the cargo config is doing the distinction by the [target family](https://doc.rust-lang.org/reference/conditional-compilation.html#target_family). I'm open to better wording